presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -470,7 +470,16 @@ public PlanNode visitJoin(JoinNode node, RewriteContext<Void> context)
             PlanNode left = context.rewrite(node.getLeft());
             PlanNode right = context.rewrite(node.getRight());
 
-            return new JoinNode(node.getId(), node.getType(), left, right, canonicalizeJoinCriteria(node.getCriteria()), node.getFilter().map(this::canonicalize), canonicalize(node.getLeftHashSymbol()), canonicalize(node.getRightHashSymbol()));
+            return new JoinNode(
```

Comment:
i changed it to multiple lines because adding the extra property made the line unreasonably long, so it didn't seem like a separate formatting change to me.  But I can split it out.